clip_benchmark/cli.py

```diff
@@ -7,37 +7,51 @@
 
 from clip_benchmark.datasets.builder import build_dataset, get_dataset_collate_fn
 from clip_benchmark.metrics import zeroshot_classification, zeroshot_retrieval, linear_probe
+from clip_benchmark.datasets import multilingual_dataset
 
 from torch.utils.data import default_collate
 
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--dataset', type=str, default="cifar10", help="Dataset to use for the benchmark")
     parser.add_argument('--split', type=str, default="test", help="Dataset split to use")
-    parser.add_argument('--model', type=str, default="ViT-B-32-quickgelu", help="Model architecture to use from OpenCLIP")
-    parser.add_argument('--pretrained', type=str, default="laion400m_e32", help="Model checkpoint name to use from OpenCLIP")
-    parser.add_argument('--task', type=str, default="zeroshot_classification", choices=["zeroshot_classification", "zeroshot_retrieval", "linear_probe"])
+    parser.add_argument('--model', type=str, default="ViT-B-32-quickgelu",
```

Comment:
would be great if you could undo all these line end changes

Reply:
Ahhh, I see my PyCharm environment is wrecking things up.